packages/client-direct/src/api.ts

```diff
@@ -183,5 +185,96 @@ export function createApiRouter(
         }
     });
 
+    router.get("/tee/agents", async (req, res) => {
+        try {
+            const allAgents = [];
+
+            for (const agentRuntime of agents.values()) {
+                const teeLogService = agentRuntime
+                    .getService<TeeLogService>(
+                    ServiceType.TEE_LOG
+                )
+                .getInstance();
+
+                const agents = await teeLogService.getAllAgents();
+                allAgents.push(...agents)
+            }
+
+            const runtime: AgentRuntime = agents.values().next().value;
+            const teeLogService = runtime.getService<TeeLogService>(ServiceType.TEE_LOG).getInstance();
+            const attestation = await teeLogService.generateAttestation(JSON.stringify(allAgents));
```

Comment:
Can the sqlite database be tampered? From my understanding, the persistent data stored in sqlite database isn't encrypted, so if it's tmapered, the application can't detect this. And application doesn't verify the correctness of the data read from sqlite databse,  This creates a security risk where an attacker could modify the database contents and the user still receive legitimate attestations. Are there any security measures in place that I might have overlooked?

Reply:
Just found the answer for the comments in README